core/rawdb/accessors_snapshot_ext.go

```diff
@@ -0,0 +1,43 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package rawdb
+
+import (
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/ethdb"
+	"github.com/ava-labs/libevm/log"
+)
+
+// ReadSnapshotBlockHash retrieves the hash of the block whose state is contained in
+// the persisted snapshot.
+func ReadSnapshotBlockHash(db ethdb.KeyValueReader) common.Hash {
+	data, _ := db.Get(snapshotBlockHashKey)
```

Comment:
Why not check and return the error?

Reply:
Totally agreed, but I think this uses the same approach as i.e. geth's `ReadSnapshotRoot` - return an empty hash in case of error. Should we leave it as is then? 🤔